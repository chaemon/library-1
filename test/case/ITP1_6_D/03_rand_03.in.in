40 38
48 4 41 14 28 8 5 17 23 25 1 26 15 37 49 24 16 49 10 3 23 8 3 2 5 9 6 37 7 3 32 34 37 10 42 20 11 36
1 48 47 25 41 18 30 26 15 33 4 44 5 20 42 22 3 1 23 28 40 20 42 17 2 40 32 25 9 42 22 42 11 12 44 16 25 24
34 6 29 23 7 16 16 42 27 31 24 34 36 17 42 48 48 7 3 20 14 20 11 5 30 39 28 3 17 15 5 10 5 3 4 11 39 19
31 4 34 36 35 16 45 15 27 26 34 0 29 2 9 1 20 45 3 47 1 18 13 33 11 30 20 15 29 41 25 17 43 47 7 20 46 16
46 37 47 1 19 43 32 28 47 46 35 14 47 2 9 45 27 2 21 13 40 17 36 23 36 17 1 40 27 31 36 17 25 47 18 38 29 25
49 21 39 4 30 1 32 23 39 19 10 40 42 12 47 4 34 17 47 17 16 36 15 47 8 17 15 11 40 20 0 0 34 35 35 27 30 31
29 23 42 12 33 38 38 18 25 6 16 28 4 31 35 37 8 44 13 42 9 0 12 49 23 7 8 17 28 39 44 46 41 36 30 28 34 20
10 35 35 19 45 37 26 5 31 16 10 10 31 48 43 29 40 42 31 49 27 39 42 31 42 47 27 8 41 21 10 42 40 17 49 8 19 10
18 6 8 37 33 18 37 32 6 12 28 23 34 28 15 38 5 2 43 8 15 1 1 16 6 8 2 30 13 13 36 34 42 28 10 20 40 26
26 3 21 19 19 14 49 3 44 38 10 0 1 39 11 43 13 7 7 5 40 17 40 22 44 25 41 16 10 31 49 1 0 10 11 1 10 47
1 11 11 42 2 42 16 20 14 11 31 5 44 44 1 37 11 41 44 14 19 27 25 27 0 36 14 31 22 22 19 47 0 16 45 22 44 13
31 0 34 4 28 17 2 30 23 26 16 25 0 37 36 47 43 4 44 49 48 25 20 44 10 6 32 11 47 43 9 32 32 46 25 40 8 28
8 20 48 11 42 4 12 1 47 5 25 48 23 16 44 11 17 6 33 0 34 13 40 7 15 0 3 30 43 22 21 34 3 8 32 22 33 11
17 34 46 46 22 40 47 39 36 41 12 19 8 47 28 0 3 39 43 27 28 46 43 30 9 13 1 41 37 46 4 41 13 33 32 33 30 22
30 42 28 10 44 23 3 28 25 4 26 21 38 16 28 4 39 40 26 11 4 23 41 40 3 14 43 47 5 16 37 4 12 30 41 30 46 21
27 3 23 29 14 31 42 48 13 20 29 39 47 49 42 45 29 27 12 39 33 25 14 16 10 45 39 8 7 1 43 16 15 29 24 28 32 15
36 15 17 46 44 16 36 38 45 27 49 29 41 26 12 3 19 20 32 35 6 31 5 31 1 11 18 43 19 22 31 30 16 34 42 10 2 34
29 43 23 37 3 15 24 13 42 21 19 20 36 16 6 1 38 47 0 2 44 22 41 9 22 38 44 27 9 16 47 34 37 6 43 3 17 47
7 15 27 6 27 9 41 1 46 43 22 31 39 49 14 41 15 42 16 2 42 32 16 49 40 21 20 43 16 6 11 6 7 36 41 14 4 45
47 4 22 30 49 13 33 43 42 17 17 12 35 22 6 30 47 11 11 18 7 39 41 20 10 20 36 46 4 13 25 0 47 49 24 4 40 42
20 21 23 25 22 11 0 39 5 45 20 19 8 28 1 11 29 35 35 0 13 20 30 22 0 29 36 44 14 49 7 37 34 22 47 11 4 27
19 14 11 0 5 44 4 45 46 18 25 28 5 25 38 32 47 45 13 16 15 0 24 18 22 26 20 14 44 11 29 37 43 46 22 20 47 36
0 24 34 36 17 43 35 25 1 25 39 32 4 36 49 12 35 47 45 20 41 9 14 18 7 4 33 28 1 23 26 42 29 11 17 44 38 40
21 13 16 20 49 19 14 3 11 47 2 3 37 42 14 1 15 7 31 8 3 39 0 28 22 11 42 17 45 19 45 49 45 16 36 43 24 3
27 26 38 26 4 37 5 45 11 12 35 31 25 27 43 35 13 36 26 20 22 29 16 46 40 39 42 45 21 8 5 31 11 38 43 44 31 16
17 2 21 24 33 6 3 35 25 2 6 20 31 33 13 6 23 28 5 29 6 38 10 16 29 49 48 15 48 20 31 33 12 40 17 49 42 14
14 49 26 9 11 7 42 0 11 1 32 9 15 19 49 13 31 42 21 13 45 33 44 21 6 8 1 49 32 21 26 23 33 10 0 44 48 19
44 18 18 40 39 10 38 17 23 8 29 19 17 5 10 3 12 16 19 6 36 37 31 15 29 7 36 41 22 5 2 8 8 48 3 21 26 26
5 37 29 27 18 47 30 44 40 32 47 11 28 25 6 47 27 1 25 12 4 37 24 17 21 43 31 35 12 35 25 1 28 14 13 4 45 2
2 24 15 10 42 7 5 10 45 30 16 28 6 33 29 30 16 43 15 40 43 47 11 4 32 32 31 49 3 7 39 26 45 43 18 47 8 47
20 41 49 19 31 9 30 25 40 0 49 19 45 15 28 10 32 47 0 10 34 47 44 18 33 34 16 12 42 25 29 48 8 37 28 2 15 27
9 40 36 26 45 18 21 12 7 41 35 4 46 17 19 44 3 6 31 42 5 44 13 36 22 24 37 38 38 46 16 2 5 28 40 21 26 12
32 25 33 24 33 22 36 47 13 26 48 42 0 29 22 11 1 21 8 8 14 4 8 24 27 48 25 5 33 23 8 46 10 44 16 45 35 4
33 34 24 4 12 21 38 34 8 6 7 43 44 38 2 31 48 5 41 25 9 41 47 7 48 49 10 28 46 16 8 40 12 49 30 8 4 19
12 24 41 16 24 25 25 34 40 7 48 29 14 3 33 27 24 7 4 43 1 31 5 13 31 48 25 8 0 3 43 35 15 33 19 22 12 22
26 45 30 43 21 9 17 23 1 47 25 37 36 2 43 48 41 19 11 47 14 15 8 4 45 43 35 26 5 44 37 30 4 27 20 47 24 23
9 6 3 9 43 18 49 43 32 8 27 10 34 14 10 39 20 34 26 4 7 21 12 28 31 0 5 35 37 41 29 22 41 47 35 4 8 39
0 25 44 30 44 47 14 37 20 36 27 39 23 21 28 42 9 2 3 16 32 41 48 48 16 12 20 37 34 38 6 43 17 11 15 26 20 3
5 21 42 39 13 0 19 38 0 10 19 37 2 23 20 4 45 32 15 34 31 25 48 40 20 45 37 23 29 49 47 14 12 19 21 18 11 27
30 45 22 37 29 9 16 33 6 31 22 35 47 43 43 23 20 45 40 30 39 16 32 0 18 25 7 13 21 35 28 31 10 41 25 25 14 6
24
18
34
14
16
6
37
33
33
48
48
20
3
15
29
33
29
27
8
14
49
23
35
43
22
27
42
41
27
41
44
11
32
42
7
38
43
30
