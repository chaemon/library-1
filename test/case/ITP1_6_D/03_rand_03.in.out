21234
26744
22820
27163
32520
26527
28381
28417
21670
25533
22687
29699
22283
30600
27200
28615
26434
27167
27886
29023
24297
28249
28647
23139
29701
24877
27400
23978
27546
29167
28895
26999
26790
24523
24176
29457
25339
28250
28553
26585
